java/org/apache/catalina/startup/LocalStrings_fr.properties

```diff
@@ -69,6 +69,7 @@ contextConfig.missingRealm=Aucun royaume (realm) n'a été configuré pour réal
 contextConfig.processAnnotationsDir.debug=Balayage du répertoire pour trouver des fichiers de classe avec annotations [{0}]
 contextConfig.processAnnotationsJar.debug=Analyse du fichier jars pour des classes annotées avec [{0}]
 contextConfig.processAnnotationsWebDir.debug=Balayage du répertoire d''applications web, pour fichiers de classe avec annotations [{0}]
+contextConfig.processAnnotationsInParallelFailure=exécution parallèle a échoué
```

Comment:
Should `exécution` be capital cased like the other messages ?

Reply:
Yes, the translation is meh. As a rule, I'd say never commit any localization IMO, just let others handle in with poeditor later.

Reply:
Thanks for the suggestion. 
Resolved with the latest commit.